lib/packages/core/src/api/factories/account/setRewardRecipient.ts

```diff
@@ -32,13 +32,20 @@ export const setRewardRecipient = (service: BurstService): (
         deadline: number,
     ): Promise<TransactionId> => {
 
-        let parameters = {
+        const parameters = {
             recipient,
             deadline: 1440,
             feeNQT: convertNumberToNQTString(parseFloat(feeNQT)),
             publicKey: senderPublicKey
         };
-        const {unsignedTransactionBytes: unsignedHexMessage} = await service.send<TransactionResponse>('setRewardRecipient', parameters);
+        const {
```

Comment:
You don't need to do this. If the Http fails, even brs returns as status 200, the error message is mapped into HttpError... Have a look at BurstService.ts